data/policy_permissions_list/master_policy_permissions_list.json

```
@@ -1516,7 +1548,11 @@
     {
       "id": "./cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt",
       "name": "Azure Hybrid Use Benefit for Windows Server",
+<<<<<<< HEAD
+      "version": "5.0",
+=======
       "version": "4.3",
+>>>>>>> master
```

Merge conflict needs resolving
```suggestion
      "version": "5.0",
```